File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int,int> countMap;
        int flag=0;
        for(int& num:nums){
            if(countMap.count(num)==0){
                countMap[num]=1;
            }
            else{
                countMap[num]++;
            }
        }
        for(int& num:nums){
            if(countMap[num]>1){
                flag++;
            }
        }
        if(flag>0){
            return true;
        }
        else{
            return false;
        }

        
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        countMap={}
        for num in nums:
            if num not in countMap:
                countMap[num]=1
            else:
                countMap[num]+=1
            if countMap[num]>len(nums)//2:
                return num
        